packages/react-components/src/components/Select/Select.css

```diff
@@ -18,9 +20,18 @@
 
 /* "(optional)" text after Label*/
 .bcds-react-aria-Select--Label > .optional {
+  padding: var(--layout-padding-none) var(--layout-padding-xsmall);
   color: var(--typography-color-secondary);
 }
 
+/* Error message */
+.bcds-react-aria-Select--Error {
+  display: flex;
```

Comment:
Is the intent here to have this element hold more than just a text string? If so, `display: flex` might be warranted, but if not, we can leave this out.

Reply:
Fixed in [6805b3f](https://github.com/bcgov/design-system/pull/390/commits/6805b3f753076d54b210f7a0eee3d5dd30ababe3)